src/Solver.h

```diff
@@ -107,6 +107,9 @@ class Solver {
   /** The number of fissionable flat source regions */
   int _num_fissionable_FSRs;
 
+  /** The number of independent transport sweeps */
+  int _num_independent_sweeps;
```

Comment:
This was just a different notation I was using with the 3D Solver. I'll remove this.